scrapper.py

```diff
@@ -25,39 +36,175 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        """
+        Extract URL
+        """
+        sections_bs = article_bs.find('div', {'class': 'sections'})
+        urls_bs = sections_bs.find_all('a')
+        for url_bs in urls_bs:
+            if len(self.urls) < self.max_articles:
+                if url_bs['href'] not in self.urls:
+                    self.urls.append(url_bs['href'])
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+
+        for seed_url in self.seed_urls:
+            response = requests.get(seed_url, headers=HEADERS)
+            if not response.ok:
+                print("Failed")
```

Comment:
this is a weak handling. imagine it failed. you just printed it to a terminal and allowed it to continue executing by creating a BS object on top of a failed request response. can you find a better and a more stable way?